### PR TITLE
install-deps.sh: add EPEL repo for non-x86_64 archs as well

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: https://tracker.ceph.com/issues/42023
Signed-off-by: Kefu Chai <kchai@redhat.com>
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: install-deps.sh [hunk 1]
```diff
@@ -367,7 +367,7 @@ else
                 if test $ID = rhel ; then
                     $SUDO yum-config-manager --enable rhel-$MAJOR_VERSION-server-optional-rpms
                 fi
-                $SUDO yum-config-manager --add-repo https://dl.fedoraproject.org/pub/epel/$MAJOR_VERSION/x86_64/
+                $SUDO yum-config-manager --add-repo https://dl.fedoraproject.org/pub/epel/$MAJOR_VERSION/$(uname -m)/
```
Comment (tchaikov) [inline install-deps.sh]:
```suggestion
                $SUDO yum install --nogpgcheck https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```
see https://fedoraproject.org/wiki/EPEL

**Comment by tchaikov** [comment]
also, i'd suggest use the prefix of `install-deps.sh`. more specific this way. `build/ops` works well as a category. but it'd be difficult to map it to a certain part of the source tree. 

and, could revise the title like, 

> add EPEL repo for non x86-64 archs as well

**Comment by smithfarm** [comment]
@tchaikov Done, thank you! Note that `ARCH` is initialized to `$(uname -m)` at the beginning of the script, but this variable was not in use. So I took this opportunity to do a little cleanup.

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by smithfarm** [comment]
```
The following tests FAILED:
	137 - mgr-dashboard-frontend-unittests (Failed)
```

**Comment by smithfarm** [comment]
@tchaikov Although the make check test as a whole did not pass, obviously it is not due to a failure in install-deps.sh

**Comment by tchaikov** [comment]
```
> ceph-dashboard@0.0.0 test:config /home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard/frontend
> if [ ! -e 'src/unit-test-configuration.ts' ]; then cp 'src/unit-test-configuration.ts.sample' 'src/unit-test-configuration.ts'; fi

...................................
  ● CephfsTabsComponent › handling of id change › should not subscribe to an new interval for the same selection

    Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.

      at mapper (node_modules/jest-jasmine2/build/queueRunner.js:25:45)

........................................................................................................................................................................................

```
jenkins test make check

**Comment by smithfarm** [comment]
jenkins test make check

**Pull request merged**